app/scripts/lib/transaction/eip5792.ts

```diff
@@ -538,7 +550,7 @@ async function getAlternateGasFeesCapability(
     if (alternateGasFees) {
       acc[chainId as Hex] = {
         alternateGasFees: {
-          supported: true,
+          supported: isSendBundleSupported,
```

Comment:
`isSendBundleSupported` can be added to condition `alternateGasFees` so we add this parameter only if it is true